motulator/grid/model/_grid_converter_system.py

```diff
@@ -0,0 +1,51 @@
+"""
+Electromechanical stiff AC grid and converter model interconnectors.
+
+This interconnects the subsystems of a converter with a grid and provides
+an interface to the solver. More complicated systems could be modeled using
+a similar template. Peak-valued complex space vectors are used.
+
+"""
+from motulator.common.model import Model
+
+
+# %%
```

Comment:
I agree with this TODO that this class could be renamed GridConverterSystem. Then, the naming would be more consistent with the naming of the drive-side system models.